### ISSUE TITLE
Gallery: Unhandled message type results in crash

### ISSUE THREAD

**Issue by litmus-ritten** [tloncorp/landscape-apps#2355]
Attempting to access `/apps/groups/groups/~fabled-faster/storage/channels/heap/~fabled-faster/texts-9123` results in the following:

```
Error: Unhandled message type: {"code":"Intro:\n \nAm, F, C...\nF, Em7, C, Em7, F\nEm7, C, Dm, F, G\n \nVerse:\n \nAm\nSomeday\n   C          Em7   F\nI know we'll meet again\n    Em7  C       Em7     F\nWithout knowing exactly when\n    G      C\nAnd until then\n     Dm           F       G\nTake care of yourself my friend\n \n                      Am\nIf you tell it to my heart\n                  F\nwhere would you start?\n           G\nWith that truth we shared\n        Am         F        G\nor the lies that drove us apart?\n \n                      Am\nIf you tell it to my heart\n                  F\nwhere would you start?\n           G\nWith that truth we shared\n        Am         F        C\nor the lies that drove us apart?\n \n                      Am\nIf you tell it to my heart\n                  F\nwhere would you start?\n           G\nWith that truth we shared\n        Am         F        G\nor the lies that drove us apart?\n \nOutro:\n \nC\nHea-a-a-art\n              G\nTell it to my heart\n \nAm, F, G, C (ad infinitum)"}
    at kn (http://localhost:8080/apps/groups/assets/index.8af1b904.js:45:71331)
    at Kl (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:19:52884)
    at mf (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:8723)
    at pf (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:947)
    at yg (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:875)
    at rn (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:730)
    at sl (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:21:10884)
    at http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:19:38716
    at e.unstable_runWithPriority (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:8:3811)
    at _r (http://localhost:8080/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:19:38494)
```

**Comment by litmus-ritten** [comment]
@urcades Relevant to your interests

**Comment by jamesacklin** [comment]
Fixed by #2318, should be out Monday morning.
